doc/sphinx_source/using/accounts.rst

```diff
@@ -0,0 +1,81 @@
+
+WHOX
+^^^^
+'WHOX <https://ircv3.net/specs/extensions/whox>`_ is a server feature that allows a client to request custom fields to be returned in a WHO response. If a server supports this capability, Eggdrop sends a WHOX query to the server when it joins a channel, allowing it to immediately determine accounts associated with channel members when Eggdrop joins a channel.
+
+Enabling Eggdrop Account Tracking
+---------------------------------
+By default, the Eggdrop config file will attempt to enable all the capabilities required for account tracking. There are two settings to pay attention to
+::
+
+  # To request the account-notify feature via CAP, set this to 1
+  set account-notify 1
+
+  # To request the extended-join feature via CAP, set this to 1
+  set extended-join 1
+
+The ability of a server to support WHOX queries is determined via a server's ISUPPORT (005) reply. If a server supports WHOX queries, Eggdrop will automatically enable this feature.
+
+Checking Account-tracking Status
+--------------------------------
+While Eggdrop is running, join the partyline and type `.status`. If account-tracking is enabled (both the server supports and Eggdrop has requested), you'll see this line
+::
+
+  Loaded module information:
+    #eggdroptest        : (not on channel)
+    Channels: #eggdroptest (trying)
+    Account tracking: Enabled           <--- This line
+    Online as: BeerBot (BeerBot)
+
+Otherwise, the prompt will tell you which required capability is missing/not enabled
+::
+
+  Loaded module information:
+    #eggdroptest        :   2 members, enforcing "+tn" (greet)
+    Channels: #eggdroptest (need ops)
+    Account tracking: Best-effort (Missing capabilities: extended-join, see .status all for details)      <---- This line
+    Online as: Eggdrop (Eggdrop)
+
+Determining if a Server Supports Account Capabilities
+-----------------------------------------------------
+A server announces the capabilities it supports via a CAP request. If you have Tcl enabled on the partyline (or via a raw message from a client), you can send `.tcl cap ls` and see if the extended-join and account-notify capabilities are supported by the server. If they are not listed, the server does not support it.
+
+A server announces if it supports WHOX via its ISUPPORT (005) announcement. If you have Tcl enabled on the partyline, you can send `.tcl issupport isset WHOX` and if it returns '1', WHOX is supported by the server.
+
+Best-Effort Account Tracking
+----------------------------
+If a server only supports some, but not all, of the required capabilities, Eggdrop will switch to 'best effort' account tracking. This means Eggdrop will update account statuses whenever it sees account information, but Eggdrop cannnot guarantee the accuracy of account association at any given time.
```

Comment:
"Eggdrop cannnot guarantee the accuracy of account association at any given time" I would say "Eggdrop cannot guarantee that the account association is up to date"